test/tensorflow_test.py

```diff
@@ -60,5 +89,15 @@ def testTensorFlowVariables(self):
 
     ray.worker.cleanup()
 
+  def testTensorFlowVariableCollision(self):
+    ray.init(num_workers=2)
+
+    ray.env.net_vars1 = ray.EnvironmentVariable(net_vars_initializer, net_vars_reinitializer)
+    ray.env.net_vars2 = ray.EnvironmentVariable(net_vars_initializer, net_vars_reinitializer)
+
+    ray.env.net_vars1.set_weights(ray.env.net_vars2.get_weights())
+
+    ray.worker.cleanup()
+
```

Comment:
Can we add another test where we create a tensorflow net on the driver (without putting it inside a reusable variable) before creating the reusable variable, so that the tensorflow variables on the driver and workers differ?